polytracker/dumptdag.py

```diff
@@ -194,7 +194,7 @@ def dump_tdag(file: Path):
 # NOTE (hbrodin): Assumes source taint was preallocated
 
 
-def gen_source_taint_used(tdagpath: Path, sourcefile: Path) -> bytearray:
```

Comment:
Why get rid of Path?

Reply:
Reverted it back. Originally I had issues with comparisons and docker and other stuff, but managed to resolve it along the way.